.github/workflows/build-and-deploy.yml

```diff
@@ -121,6 +121,53 @@ jobs:
           maven-central-password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
           signing-key: ${{ secrets.SIGNING_KEY }}
           signing-password: ${{ secrets.SIGNING_PASSWORD }}
+  test-multi-stage-deployment:
+    runs-on: ubuntu-latest
+    if: >-
+      !github.event.repository.fork
+      && (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository)
+    steps:
+      - name: Compute the version of the target test project
+        id: versiontrick
+        shell: bash
+        run: |
+          # Idea: the regex matcher of Renovate keeps this string up to date automatically
+          # The version is extracted and used to access the correct version of the scripts
+          USES=$(cat <<TRICK_RENOVATE
+          - uses: DanySK/template-for-kotlin-multiplatform-projects@0.1.7
+          TRICK_RENOVATE
+          )
+          echo "Scripts update line: \"$USES\""
+          echo "Computed version: \"${USES#*@}\""
+          echo "version=${USES#*@}" >> $GITHUB_OUTPUT
+      - name: Checkout Template-for-Kotlin-Multiplatform-Projects ${{ steps.versiontrick.outputs.version }}
+        uses: actions/checkout@v3.3.0
+        with:
+          fetch-depth: '0'
+          path: 'kt-mp-multi-stage'
+          ref: "${{ steps.versiontrick.outputs.version }}"
+          repository: 'DanySK/Template-for-Kotlin-Multiplatform-Projects'
+          submodules: 'recursive'
+      - name: Checkout publish-on-central
+        uses: actions/checkout@v3.3.0
+        with:
+          path: 'publish-on-central'
+      - name: Dry-deploy
+        uses: DanySK/build-check-deploy-gradle-action@2.1.21
+        with:
+          build-command: true
+          check-command: true
+          deploy-command: |
+            COMMAND='./gradlew --include-build ../publish-on-central createStagingRepositoryOnMavenCentral --parallel'
+            $(echo "$COMMAND") || $(echo "$COMMAND") || $(echo "$COMMAND")
+            REPO_IDS='cat build/staging-repo-ids.properties'
+            $(echo "$REPO_IDS") || $(echo "$REPO_IDS") || $(echo "$REPO_IDS")
```

Comment:
uhmm... I do not understand the test. It should be something like `[ -e `build/staging-repo-ids.properties ]`?
Also maybe the contents should be checked with a regex

Reply:
Yes, the original test was too weak...
Fixed checking if the file exists and match its content with a regex, as you suggested.
Let me know if the new test could be effective